tests/monotouch-test/Foundation/UrlSessionTaskTransactionMetricsTest.cs

```diff
@@ -58,12 +58,7 @@ public void Properties ()
 					Assert.Null (sttm.ResponseEndDate, "ResponseEndDate");
 					Assert.Null (sttm.ResponseStartDate, "ResponseStartDate");
 				}
-#if __IOS__
-				if (TestRuntime.CheckXcodeVersion (11, 6)) // Returns true in iOS 13.6
-					Assert.True (sttm.ReusedConnection, "ReusedConnection");
-				else
-#endif
-					Assert.False (sttm.ReusedConnection, "ReusedConnection");
+				Assert.That (sttm.ReusedConnection, Is.EqualTo (true).Or.EqualTo (false), "ReusedConnection");
```

Comment:
Why do we need to check this value if it passes regardless of true or false? Is it a nullable bool perhaps and we want to make sure its not null?

Reply:
The return type is not a nullable, so it is not a bool?. Which means that we need to assert that the property works, but we do not care about the value, and we cant check against null. How do we do that with a value type and ignore the value? We could have done this Assert.True (true || sttm.ReusedConnection, "ReusedConnection") but I think that the way @rolfbjarne wrote it is more explicit.

Reply:
Thanks!